db/version_set.cc

```diff
@@ -3423,6 +3425,7 @@ std::string Version::DebugString(bool hex, bool print_stats) const {
       AppendNumberTo(&r, files[i]->fd.largest_seqno);
       r.append("]");
       r.append("[");
+      // TODO(laiyingchun): Pegasus data DebugString() not work correctly now.
```

Comment:
Formerly I considered that keys are encoded in Pegasus format, DebugString the keys is not readable, but now I think this TODO comment is redundant, because keys could be print in hex mode. I'll remove it.